prov/efa/src/rdm/efa_rdm_pke.c

```diff
@@ -614,43 +614,52 @@ int efa_rdm_pke_write(struct efa_rdm_pke *pkt_entry)
  *
  * @param[in] pke_vec	packet entries that contains information of receive buffer
  * @param[in] pke_cnt	Number of packet entries to post receive requests for
+ * @param[in] flags  	user supplied flags passed to fi_recv
  * @return		0 on success
  * 			On error, a negative value corresponding to fabric errno
  */
 ssize_t efa_rdm_pke_recvv(struct efa_rdm_pke **pke_vec,
-			  int pke_cnt)
+			  int pke_cnt, uint64_t flags)
 {
 	struct efa_rdm_ep *ep;
 	struct ibv_recv_wr *bad_wr;
 	struct ibv_qp *qp;
 	int i, err;
+	size_t wr_index;
 
 	assert(pke_cnt);
 
 	ep = pke_vec[0]->ep;
 	assert(ep);
 
+	wr_index = ep->base_ep.efa_recv_wr_index;
+
 	for (i = 0; i < pke_cnt; ++i) {
```

Comment:
Another approach may be having `efa_rdm_ep_post_internal_rx_pkts` call this function in a loop of `pkt_cnt` times, where the first `pkt_cnt-1` times are with FI_MORE.

But this version should be good as well... We can revisit it later if necessary.